ansible/pbTestScripts/testJDK.sh

```diff
@@ -21,13 +21,17 @@ mkdir -p $HOME/testLocation
 [ ! -d $HOME/testLocation/aqa-tests ] && git clone https://github.com/adoptium/aqa-tests.git $HOME/testLocation/aqa-tests
 # cd to aqa-tests as required by https://github.com/adoptium/aqa-tests/issues/2691#issue-932959102
 cd $HOME/testLocation/aqa-tests
+if [[ "$(uname)" == "SunOS" ]]; then
+	export PATH=/usr/local/bin:$PATH
+fi
 $HOME/testLocation/aqa-tests/get.sh
 cd $HOME/testLocation/aqa-tests/TKG || exit 1
 
 # Solaris runs a different test to Linux.
 # See: https://adoptium.slack.com/archives/C53GHCXL4/p1641311568115100?thread_ts=1641296204.114900&cid=C53GHCXL4
 if [[ "$(uname)" == "SunOS" ]]; then
 	export BUILD_LIST=system
+	export PATH=/usr/local/bin:$PATH
```

Comment:
Isn't this superfluous given that it is set on SunOS in the earlier if statement which would still be active here?

Reply:
Fixed!